examples/jsm/animation/CCDIKSolver.js

```diff
@@ -283,7 +283,7 @@ function setPositionOfBoneToAttributeArray( array, index, bone, matrixWorldInv )
  */
 class CCDIKHelper extends Object3D {
 
-	constructor( mesh, iks = [] ) {
+	constructor( mesh, iks = [], options = { sphereSize: .25 } ) {
```

Comment:
I had to change the CCDIKHelper sphere size in my scene, so I made this an optional parameter (no breaking change)

Reply:
One last thing: Let's not introduce an `options` parameter. We don't do this since a while anymore. Let's make the parameter explicit like with other helpers. So the signature should be:
```js
constructor( mesh, iks = [], sphereSize = 0.25 ) {

Reply:
I'll update the code 👍 .